### PR TITLE
updated v3.9.15

### PR DESCRIPTION
**Pull request by mbargull**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mbargull** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
:tada: looks all green

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
         uses: actions/checkout@v3
       - name: automerge-action
         id: automerge-action
-        uses: conda-forge/automerge-action@main
+        uses: conda-forge/automerge-action@master
```
[the remark this answers is not in the document]
Comment (mbargull) [reply .github/workflows/automerge.yml:16]: Yes, but it's not important. See https://github.com/conda-forge/python-feedstock/pull/604#discussion_r1028091977

**Comment by bollwyvl** [comment]
@conda-forge/python looks like _one_ upload failed :crying_cat_face: 

https://github.com/conda-forge/python-feedstock/runs/9638099160

**Pull request merged**